tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Show double feature - buglist shows on focus, hides on blur": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("description")
+          .click()
+          .end()
+          .sleep(9500)
```

Comment:
This is a super long delay, are we just waiting until `bugList` is shown? By default all `find*` commands will wait for the DOM element to exist before failing (as long as it happens before the find timeout). Does it work without this `sleep` call?

Reply:
The delay was just for testing as there is a bug anyway... 

The DOM element is always there (so finding the DOM element "works" with and without `sleep`). The issue here is, the DOM element with the ID `#bugList` is filled with list items (the items are related issues). When the input loses focus, the list is removed again. 

![duolicate_issue](https://user-images.githubusercontent.com/562354/34208959-9f11ebae-e590-11e7-9fe3-be3371d759bb.gif)

The issue is that in the test, the list items still exist.

I thought that `.end()` would dismiss the scope it has before and the next element you'd look for (e.g. `findById('description')`) would be grabbed "fresh", but the test seems to get an "old" state? (I hope, this was kind of understandable.)